### DISCUSSION TITLE
Practica CSS

### DISCUSSION THREAD

**Discussion by zarry0** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#435] [Show and tell]
Les muestro como me quedó mi practica de CSS y aparte el link de mi repo por si alguien gusta usarlo como guía.

https://paginavacunacionlx.netlify.app/

https://github.com/zarry0/frontEnd_LaunchX/tree/main/CSS/pagina_vacunacion
![pagina_vacunacion](https://user-images.githubusercontent.com/87690107/158738272-36b93270-a2ff-4d23-b442-0bd87b767a5c.png)

**Comment by MauricioNovelo** [comment]
Muy buen repositorio!
